src/registrar/fixtures/fixtures_users.py

```diff
@@ -191,6 +191,13 @@ class UserFixture:
             "email": "kim+dotgov@truss.works",
             "title": "Farmer",
         },
+        {
+            "username": "0f784268-a481-445e-9e37-ea2be43ce318",
+            "first_name": "Daisy",
+            "last_name": "Gutierrez",
+            "email": "dgutierrez@guydo.com",
```

Comment:
Is this a personal email? It soundes like Abe and Aditi were going to be able to use `+` emails through ecs soon- maybe you could use that as well?

Reply:
No, it's company email. I didn't want to have to deal with IT support tbh haha